test/ngAnimate/animateCssDriverSpec.js

```diff
@@ -103,6 +103,14 @@ describe("ngAnimate $$animateCssDriver", function() {
         driver({ element: element });
         expect(capturedAnimation[1].applyClassesEarly).toBeFalsy();
       }));
+
+      it("should not provide a method into the $animateCss animation if the animation is not structural", inject(function() {
```

Comment:
it should not provide a `method` as an option value if the animation is not structural